optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -942,40 +943,110 @@ def forward(
         - add new arg flash_attention_causal_mask
         - add new arg flash_attention_fast_softmax
         """
-        residual = hidden_states
+        if attn_batch_split > 1 and past_key_value is None:
+            print(" ##########################   PRROMPT")
```

Comment:
can you please remove this ```print```? it seems to be for debug

Reply:
Done